### ISSUE TITLE
Upgrading from stable 2.0.0.0 to 2.0.0.2 improve host share performance by nearly 10x ! Why ?

### ISSUE THREAD

**Issue by Yivan** [docker/for-mac#3492]
Hello,

We just upgraded on a MacOS Docker for Mac from version 2.0.0.0 to 2.0.0.2.
We use symfony application, with PHP code on a bind mount on the OS.
**Ususally each request need at least 1 seconde** before replying (HTTP Time to first response). Because of the sharing code files with the host.
Even with cache activated, it takes 1 seconde. Here a sample of our docker-compose.yml:

```
...
        volumes:
            - "${PROJECT_DIRECTORY}:/var/www/html:cached"
...
```

We just upgraded to the last stable 2.0.0.2, and the mean time of **each request is now 170 milliseconds** !!!
Ouch !! this is a wonderful improvement from 1 seconde to 170 ms ! Nearly native performance. With a symfony apps sending many ajax requests (each one was taking 1 sec before, and now just 170 ms), it run fast now.

We try to find any information on this realease to have the technical details which made this possible, but nothing about performance changes.
Desactivating experimental features keeps this performance improvement, so it is not linked to experimental features.

So this is not an issue for a bug, but more an issue to understand why such a big improvement as it a really very very good things !
On Windows, same docker version (2.0.0.2), we have always 1 seconde latency...

I hope some other people/docker team can explain this and confirm they had the same experience.
Maybe it should be putting in big news beacuse no information everywhere about this.

Best regards.

**Comment by williamkimswe** [comment]
Hi! 

We have the same kind of performance issues with slow loading, so I'm kind of curious about the version you are referring to 2.0.0.2. The latest version for mac is 2.0.1.0? 

Best regards,

**Comment by williamkimswe** [comment]
Sorry, I just realized I'm running the Edge version and that one has another version tree. 

Cheers

**Comment by taddis** [comment]
Sadly, I am not able to reproduce those stats (far from it) with v2.0.0.2.  :/

**Comment by Yivan** [comment]
Hi, thanks all for your answer.

We made some further test on other computers with MacOS. For the same project/docker config, they are fast even on previous version of Docker for Mac.
So it seems, **the speed up we saw was not linked to the Docker upgrade, but to some other things on this computer**. Sorry for the annoyance.

We made tests on three MacOS computers, and all have really good performance with Symfony (we put the tmp directory of symfony directly on the container, not on the shared folder to improve performance).
Depending power of the computer we get time request between 150-250 ms, which is good.
We made tests on Docker for Windows on some others Windows 10 computer, and all have time request of 1 seconde on exactly the same project/docker config than the MacOS.

So to conclude this thread, Docker 2.0.0.2 does not improve performance,
but from our tests we found MacOS is really better than Windows in terms of performance when using binded folder with the hosts.
I think MacOS is more performant because of the cache system (`:cache` on the volume bind mounts) and use osxfs (`https://docs.docker.com/docker-for-mac/osxfs/`).

I close this issue.
